pkgs/by-name/bu/bundler/package.nix

```diff
@@ -20,28 +20,10 @@ buildRubyGem rec {
     substituteInPlace $out/bin/bundle --replace-fail "activate_bin_path" "bin_path"
   '';
 
-  nativeInstallCheckInputs = [
-    versionCheckHook
```

Comment:
Having booth the versionCheckHook and passthru.tests.version is kinda pointless but why remove both? Why not keep the versionCheckHook ?